### ISSUE TITLE
Spell checker

### ISSUE THREAD

**Issue by tmoemke** [textext/textext#128]
Adding support for ispell, aspell or something similar would be very useful.

**Comment by jcwinkler** [comment]
It may be a little bit off topic but do you process such ammounts of text in TexText that a spell checker is really needed? What is your use-case? I am just asking for interest.

The ability to spell-check would introduce an additional dependency, especially on Windows. I am not really shure if we should open that door...

**Comment by jcwinkler** [comment]
(OK, aspell is installed together with Inkscape on Windows, so this is not a blocker as initially thought)

**Comment by cdmask** [comment]
Don't really think it is necessary.

**Comment by tmoemke** [comment]
Thanks for your quick response! I am really happy that you consider my suggestions.

I am a scientist and a few years ago I started to use inkscape/JessyInk for all of my presentations. Until now, this is the only possibility I found to combine a powerful vector graphics program with LaTex (using textext). Therefore textext has a key role in my workflow.

I do not only write the formulas using textext, but all the text appearing on the slides. This way, there is no change of font, I can mix math with normal text, and for me writing latex is the most convenient way to write and format text.

Therefore I am very thankful for your work on the extension. A spell checker is something I had a long time on my personal wish-list for the extension, but previously I did not consider to write a feature request, since after all this might be a quite specific use case. My last talk (a job talk), however, had an embarrassing and easily preventable typo. I therefore thought that it might be worth to give it a try and to write a feature request.

Best regards,
Tobias Moemke
